makezip.sh

```diff
@@ -13,10 +13,10 @@ then
 	echo "PyFFI test release - documentation not included" > docs/index.html
 else
 	rm -rf docs
-	pushd docs-sphinx
+	pushd docs
 	make html SPHINXBUILD=sphinx-build-3.2
```

Comment:
Is this still correct since you've bumped versions.

Reply:
I didn't bump versions though... also, makezip.bat and makezip.sh use epydoc. So I left that un touched in the new branch.